spec/vc.md

```diff
@@ -0,0 +1,80 @@
+| `statusListCredential` | String       | Yes      | A URL which uniquely identifies a verifiable credential whose type is `StatusList2021Credential`. |
+
+**Additional Notes:**
+- When representing Status List Credentials, as opposed to including a status in another VC, the Status List Credential must [follow the guidance here](https://www.w3.org/community/reports/credentials/CG-FINAL-vc-status-list-2021-20230102/#statuslist2021credential).
+
+### Credential Schema Data Model
+
+Following from [this data model](https://w3c.github.io/vc-json-schema/#jsonschema).
+
+| Property        | JSON Representation | Required | Notes          |
+| --------------- | ------------------- | -------- | -------------- |
+| `id`            | String              | Yes      | A URL which uniquely identifies the JSON Schema for the associated Verifiable Credential. |
+| `type`          | String              | Yes      | Must be set to `JsonSchema`. |
+
+**Additional Notes:**
+- Though [the referenced spec](https://w3c.github.io/vc-json-schema/) is designed for v2 of the VC Data Model, we are using it anyway since there is no standard way to use the `credentialSchema` property in v1.1 of the data model.
+
+## Verifiable Presentation Data Model
+
+Following from [this guidance](https://www.w3.org/TR/vc-data-model/#presentations-0), which extends on the data model above.
+
+| Property      | JSON Representation | Required | Notes          |
+| ------------- | ------------------- | -------- | -------------- |
+| `@context`    | Array of strings    | Yes      | Contexts defining the meaning of terms within the presentation. Must include at least `"https://www.w3.org/2018/credentials/v1"`. |
+| `id`          | String              | Yes      | A URI representing a unique identifier for the presentation. Recommended to be of form `urn:uuid:3978344f-8596-4c3a-a978-8fcaba3903c5`. |
+| `type`        | Array of strings    | Yes      | Type(s) of the presentation. Must include `VerifiablePresentation`. |
+| `holder`      | String              | Yes      | A DID representing a unique identifier for the entity that created the presentation. |
+| `issuanceDate`| String              | Yes      | [XML Datetime](https://www.w3.org/TR/xmlschema11-2/#dateTime) value for when the presentation was created. |
+| `expirationDate` | String           | No       | [XML Datetime](https://www.w3.org/TR/xmlschema11-2/#dateTime) value after which the presentation is no longer valid. |
+| `verifiableCredentials` | Array of strings | Yes    | An array with at least one value, containing the JWT representation of [Verifiable Credential](#verifiable-credential-data-model) objects. |
+
+**Additional Notes:**
+- No [JSON-LD processing](https://www.w3.org/TR/vc-data-model/#json-ld) is performed.
+- Embedded proofs, using the `proof` property must not be present.
+- The `type` property may only contain `VerifiablePresentation` or the URI of a JSON Schema, if one is used for the presentation.
```

Comment:
ya a bit of confusion:

```
{
  "@context": [
    "https://www.w3.org/2018/credentials/v1",
    "https://www.w3.org/2018/credentials/examples/v1"
  ],
  "id": "urn:uuid:3978344f-8596-4c3a-a978-8fcaba3903c5",
  "type": ["VerifiablePresentation", "CredentialManagerPresentation"],
  "verifiableCredential": [{  }],
  "proof": [{  }]
}

```

in the spec they have an extra type just added in.

Are you saying we are drawing a hard line and only adding a type if it is a URI of a JSON Schema?

Will we accept creds that just have a random type like the example?

Reply:
> Are you saying we are drawing a hard line and only adding a type if it is a URI of a JSON Schema?
this

> Will we accept creds that just have a random type like the example?

it's not random in this example because the term is defined in the provided example context
because we're not doing json ld processing we should not have a type like this

if we use a json schema we can include it here, .... or just include it in the `credentialSchema` property

Reply:
updated the language in both places